dist/backdraft.js

```diff
@@ -575,8 +581,101 @@ _.extend(Plugin.factory, {
         return columnType;
       }
     });
+  },
+
+  _applySavedState: function() {
+    var reportSettings = this.table.options.reportSettings;
```

Comment:
agreed (if we keep this functionality down in backdraft at all).  I like `persistedDataTableSettings`..?